docs/getting-started/handling-errors.md

```diff
@@ -146,8 +146,8 @@ app.get('/fruit/:name', async (request, response, next) => {
         if (error.code === 'ERR_ASSERTION') {
             return next(new HttpError({
                 statusCode: 400,
-                code: 'INVALID_FRUIT_NAME',
-                message: 'Please provide a fruit name'
+                code: 'FRUIT_NAME_NOT_PROVIDED',
```

Comment:
The error is thrown if the string is empty, while `INVALID_FRUIT_NAME` suggests that a string could have been provided but isn't a fruit (e.g. `'elephant'`), though this is only validated by the subsequent API call.

`FRUIT_NAME_NOT_PROVIDED` is more specific in detailing that a non-whitespace string but not provided whatsoever.